frontend/src/pages/analytics/data-management/AnalyticsDataManagement.tsx

```diff
@@ -55,6 +62,41 @@ const AnalyticsDataManagement: React.FC = () => {
     },
   ];
 
+  const onClickTriggerScan = async () => {
+    try {
+      const { success }: ApiResponse<any> = await triggerScan(
+        defaultStr(projectId),
+        defaultStr(appId)
+      );
+      if (success) {
+        alertMsg(t('analytics:metadata.scanTriggeredSuccessfully'), 'success');
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
Should we show the validation errors, for example, 429 xxx?

Reply:
add new error msg handler for 429.